### ISSUE TITLE
Ping The Servers

### ISSUE THREAD

**Issue by aaronsace** [aaronsace/MultiPoolMiner#528]
Get rid of region and ping the servers instead.

**Comment by fonyo** [comment]
Does this also mean removing the choice from the equation? What about fallback and what happens when a currently pinged/used server goes down?

**Comment by aaronsace** [comment]
Yes, in the end, the point of selecting the region is to getting the lowest latency anyway; just rank them 1 to 10 instead of region names. 

It could double up as detecting pools going down as well by setting the profit rank to zero.

**Comment by fonyo** [comment]
So they are pinged and evaluated in each script loop, aren't they?

Remember when someone asked whether they cannot mine from somewhere like Africa because it wasn't listed?

Hmm, removing human errors. Hmm, I like it!

Instead of replace you could add "-region auto" then you actually feature enhance and keep choice intact for advanced users.
